src/lib/OpenEXR/ImfZstdCompressor.cpp

```diff
@@ -0,0 +1,123 @@
+//
```

Comment:
Please run clang-format on this file with `-style=file`.

Reply:
done